feature/system/gnmi/metadata/tests/large_set_consistency_test/large_set_consistency_test.go

```diff
@@ -227,7 +227,12 @@ func testLargeMetadata(t *testing.T, gnmiClient gpb.GNMIClient, dut *ondatra.DUT
 	}
 	// Encode the bytes to a base64 string.
 	largeMetadata := base64.StdEncoding.EncodeToString(randomBytes)
-	largeMetadata = largeMetadata[:size]
+	if deviations.MetadataSizeAdjustRequired(dut) {
+		// reduce 25% of size to accomodate Encoding overhead
```

Comment:
what's the encoding overhead?

this doesn't seem to fit within the deviation guidelines 

cc @robshakir